docs/visual/e2e-testing/quickstart.md

```diff
@@ -0,0 +1,227 @@
+---
+id: quickstart
+title: Visual E2E Quickstart
+sidebar_label: Quickstart
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+Get up and running quickly on Visual E2E with a simple automated test using our demo website, [Swag Labs](http://saucedemo.com).  
+
+
+## What You'll Need
+
+* A [Sauce Labs self-serve or enterprise account](https://saucelabs.com/pricing). Visual Testing is not available for free trials. Please [contact us](https://saucelabs.com/contact) if you have any questions.
+* Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings).
+* Your [Screener API Key](https://screener.io/v2/account/api-key).
+* A VPN connection.
+* A GitHub account.
+* Have Node.js installed.
+* Optional: have Mocha and/or Jest installed.
+
+
+## Set Up Environment
+
+1. Establish a VPN connection on your local machine.
+1. From your terminal, clone the [Visual E2E demo repository](https://github.com/luishernandezv/visual-e2e) to your machine:
+  ```bash
+  git clone https://github.com/luishernandezv/visual-e2e
```

Comment:
@kimsauce we shouldnt use @luisVelasquezSauce personal repo on our docs. We should defer to the examples created and owned by SA/SE. We should talk about getting the easiest example under github/saucelabs

Reply:
@vrunoa True - it would be good if we could move his examples there. He created this repo specifically for this doc. Our Quickstart docs need to get a new user up and running with a product in ~10-15 mins, just to see how it works. Luis' examples are perfect bc they're so straightforward.

Reply:
@kimsauce what about using this example? https://github.com/saucelabs-training/demo-js/blob/main/webdriverio/webdriver/examples/visual-e2e/README.md
instructions are pretty straightforward

Reply:
Just created a ticket for this --> https://saucedev.atlassian.net/browse/TCD-2174